storagenode/vouchers/vouchers_test.go

```diff
@@ -0,0 +1,204 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package vouchers_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/golang/protobuf/ptypes"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/internal/testidentity"
+	"storj.io/storj/internal/testplanet"
+	"storj.io/storj/internal/teststorj"
+	"storj.io/storj/pkg/auth/signing"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/storagenode"
+	"storj.io/storj/storagenode/storagenodedb/storagenodedbtest"
+)
+
+func TestVouchersDB(t *testing.T) {
+	storagenodedbtest.Run(t, func(t *testing.T, db storagenode.DB) {
+		ctx := testcontext.New(t)
+		defer ctx.Cleanup()
+
+		vdb := db.Vouchers()
+
+		satellite := testidentity.MustPregeneratedSignedIdentity(0, storj.LatestIDVersion())
+		storagenode := testidentity.MustPregeneratedSignedIdentity(1, storj.LatestIDVersion())
+
+		expiration, err := ptypes.TimestampProto(time.Now().UTC().Add(24 * time.Hour))
+		require.NoError(t, err)
+
+		voucher := &pb.Voucher{
+			SatelliteId:   satellite.ID,
+			StorageNodeId: storagenode.ID,
+			Expiration:    expiration,
+		}
+
+		// GetValid with no entry
```

Comment:
Could we maybe elaborate more on these test case descriptions? Like "Tests that GetValid returns nil when there is no voucher corresponding to satellite.ID"